components/molecules/BenefitCardHeaderSummary.js

```diff
@@ -0,0 +1,33 @@
+import propTypes from 'prop-types'
+import { SummaryTypes } from '../../objects/UniversalBenefit'
+
+export default function BenefitCardHeaderSummary(props) {
+  const t = props.locale
+  let typesWithLinks = [SummaryTypes.PaymentAmount, SummaryTypes.LatestStatus]
+
+  return (
+    <div>
+      <p>{t[props.summary.type].title}</p>
```

Comment:
Maybe this should be a head since it's a title? `<h2>` or something like that

Reply:
Hmmm maybe, it would have the most effect on a screen reader, and I'm not sure it's actually desired based on the rest of the card (the task group have titles that are a header). Either way this is something for PO & Design team to talk about with. 
Since it's currently a `p` in the old benefitCard it's best to leave it for now.

Reply:
nvm I agree it should be a header of some kind for a11y - but I think maybe these "summaries" need to be under their own header - which isn't in the design. Still something to raise with them.

Reply:
and I'm correcting myself again, we can't make them headers outright because they would be repeated on the page

Reply:
Fair enough! :)